TRN-151/Database.php

```diff
@@ -0,0 +1,89 @@
+<?php
+  namespace FormData;
+  class Database{
+    //set connection variable
+    public $conn;
+    function __construct($conn){
+      $this->conn = $conn;
+    }
+    //function to send the data into database
+    function sendData($table1data,$table2data,$table3data){
+      //$table#data is an array which has data about the of table
+      //Query to insert in employee_code_table
+      $query = "INSERT INTO employee_code_table (employee_code, employee_code_name, employee_domain)
+      VALUES ('".$table1data[0]."', '".$table1data[1]."', '".$table1data[2]."');";
```

Comment:
Read about mysqli prepared statements, and try to implement it here. Read about the advantages of using prepared statement over this.